packages/editor-ui/src/components/mixins/genericHelpers.ts

```diff
@@ -77,7 +77,7 @@ export const genericHelpers = mixins(showMessage).extend({
 		async callDebounced (...inputParameters: any[]): Promise<void> { // tslint:disable-line:no-any
 			const functionName = inputParameters.shift() as string;
 			const debounceTime = inputParameters.shift() as number;
-			const trailing = inputParameters.includes(true);
+			const trailing = inputParameters.some(element => element['trailing']);
```

Comment:
can we fix this to be more specific? instead of using an array function? for example
```
		 const functionName = inputParameters[0] as string;
			const debounceTime = inputParameters[1] as number;
			const trailing = inputParameters[2]; as boolean
```

Reply:
Sure. But I think in the code, somewhere we call this function like callDebounce(funcName, debouncTime, boolean), and that boolean is not related to trailing. But if you prefer this way, I will update it like you suggest.

Reply:
For example in NodeView we have something like this: ```this.callDebounced('nodeSelectedByName', 100, connections.main[0][0].node, false, true);```. If we go that way, it will affect this function and other as well. And this booleans there are note related to trailing/leading since we introduce this with the templates.